whole-body-controller-core/robot-arm-three/build.gradle.kts

```diff
@@ -13,12 +13,12 @@ ihmc {
 }
 
 mainDependencies {
-   api("us.ihmc:simulation-construction-set:0.22.10")
    api("us.ihmc:euclid:0.19.1")
    api("us.ihmc:simulation-construction-set-tools:0.14.0-230207")
```

Comment:
See if that dependency is still necessary

Reply:
Nope, removed